lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/NestedJourneyTypes.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.ipv.core.processjourneyevent.statemachine;
+
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@Getter
+@ExcludeFromGeneratedCoverageReport
+public enum NestedJourneyTypes {
+    // When a new nested-journey is added, it needs to be added here
+    // as the statemachine will look here when initialising
+    ADDRESS_AND_FRAUD("address-and-fraud"),
+    KBVS("kbvs"),
+    STRATEGIC_APP_TRIAGE("strategic-app-triage"),
+    WEB_DL_OR_PASSPORT("web-dl-or-passport");
+
+    private final String journeyName;
+
+    NestedJourneyTypes(String journeyName) {
+        this.journeyName = journeyName;
+    }
+}
```

Comment:
I tried some other ways which meant we didn't have to define this `NestedJourneyTypes` enum and we could just iterate through all the files in `nested-journeys` folder but they were quite messy and also had to iterate through all files in the resource folder instead of just looking down a subfolder (I don't think there's a way built-in way to list all files with `getResource` :')). 

Though I can see how maintaining this file (along with the constants in the journey-map vis can be annoying though :/

Reply:
I think it's not too bad - we have to do something similar when adding a new main journey type as well.